templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,593 @@
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Glance</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Glance is an image service, responsible for uploading, managing and retrieving cloud images for instances running on OpenStack. Glance works across a variety of stores to provide the most convenient location of images for organisations. </p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Neutron</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Neutron provides network connectivity between OpenStack instances, enabling multi-VM deployments. For this purpose, Neutron uses various software defined networking (SDN) technologies, including Open Virtual Network (OVN), Open vSwitch (OVS), Juniper Contrail, Cisco ACI, etc. </p>
+    </article>
+  </div>
+  <div class="row u-equal-height">
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Cinder</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Cinder is a storage component that is responsible for provisioning, management and termination of persistent block devices. Those can be later attached to the instances running on OpenStack to enable persistent block storage.</p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Swift</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Swift is another storage component that provides a highly available and scalable object storage service similar to Amazon S3. It enables storing and retrieving unstructured data objects using a RESTful API for both OpenStack services and instances running on the cloud.</p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Keystone</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Keystone serves as an identity service, providing authentication and authorization functions for the users in order to enable multi-tenancy. Keystone can be easily integrated with external identity systems, such as lightweight directory access protocol (LDAP) or Active Directory.</p>
+    </article>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row p-logo-section">
+    <div class="col-8">
+      <h2>Companies involved in OpenStack</h2>
+      <p>OpenStack was originally launched in July 2010 by Rackspace and NASA as an open source initiative that combined NASA’s Nebula platform and Rackspace’s Cloud Files platform.</p>
+      <p>Today, hundreds of companies contribute to OpenStack code, with many more playing a part in the broader community.</p>
+    </div>
+    <div class="col-12 p-logo-section__items">
```

Comment:
this col-12 is in the same row as the col-8. Maximum cols per row is 12. Can you not use `u-fixed-width`?